packages/open-next/src/utils/binary.ts

```diff
@@ -60,8 +60,9 @@ const commonBinaryMimeTypes = new Set([
 ]);
 
 export function isBinaryContentType(contentType?: string | null) {
-  if (!contentType) return false;
+  if (contentType == null) return false;
 
-  const value = contentType?.split(";")[0] ?? "";
+  const value = contentType.split(";").at(0);
+  if (value == null) return false;
```

Comment:
can value ever be `== null` ?

Reply:
if array length is 0, .at(0) will return undefined, and value == null will be true.